### PR TITLE
Enable asyc bootsrap file

### PR DESCRIPTION
**Pull request by abachar**

### CODE REVIEW

**Comment by hassen-yahyaoui** [comment]
+1

**Comment by igorlovich** [comment]
+1

**Comment by DavertMik** [comment]
:+1:

File: lib/codecept.js [hunk 1]
```diff
@@ -45,13 +45,18 @@ class Codecept {
...
-    this.bootstrap();
+    this.bootstrap(callback);
   }
 
   // loading bootstrap
-  bootstrap() {
+  bootstrap(callback) {
     if (this.config.bootstrap && fileExists(fsPath.join(codecept_dir, this.config.bootstrap))) {
-      require(fsPath.join(codecept_dir, this.config.bootstrap));
+      var bootsrap = require(fsPath.join(codecept_dir, this.config.bootstrap));
```
Comment (DavertMik) [inline lib/codecept.js]: boot**srap**

File: lib/codecept.js [hunk 2]
```diff
@@ -45,13 +45,18 @@ class Codecept {
...
+      if (typeof bootsrap === 'function') {
+          bootsrap(callback);
+      } else {
```
Comment (DavertMik) [inline lib/codecept.js]: don't do else, just return

File: lib/command/definitions.js [hunk 3]
```diff
@@ -36,42 +36,46 @@ module.exports = function (genPath) {
...
-        methodsOfObject(helper).forEach((action) => {
-            let actionAlias = container.translation() ? container.translation().actionAliasFor(action) : action;
-            if (!actions[actionAlias]) {
-                let params = getParamNames(helper[action]);
-                if (params) params = params.join(', ');
-                if (!params) params = '';
-                methods.push(`    ${(actionAlias)}: (${params}) => any; \n`);
-                actions[actionAlias] = 1
+    codecept.init(testsPath, function(err) {
+      if (!err) {
```
Comment (DavertMik) [inline lib/command/definitions.js]: please don't add another nesting level this way. Probably here you can do

```js
if (err) {
   // maybe we should do smth with err here?
  return;
}
// continue basic flow
```

**Pull request merged**